### ISSUE TITLE
Perm link - desired

### ISSUE THREAD

**Issue by HubMiner** [opening-hours/opening_hours-statistics#17]
Please consider implementing functionality for permanent link which would save specific selections and time range.

Definition of warning is not clear, is that same as error?

**Comment by ypid** [comment]
Good idea. I will consider it when I come to reworking the stats which will take a while.

> Definition of warning is not clear, is that same as error?

Same as in https://github.com/opening-hours/opening_hours.js/issues/290#issuecomment-502399001. There are warnings "parsed with issues" (warning values in the stats) and there are errors "failed to parse" (opposite of parsed values).

**Comment by HubMiner** [comment]
Then can we have a separate check box just for errors (submit as new issue?)

**Comment by ypid** [comment]
> submit as new issue?

Good idea, lets keep this clean. I opened #18 for this.
